torch/_inductor/overrides.py

```diff
@@ -587,7 +645,46 @@ def fuse_conv_bn(gm: torch.fx.GraphModule, inplace=False):
                 node.replace_all_uses_with(node.args[0])
                 gm.graph.erase_node(node)
                 gm.graph.lint()
+    for pattern in module_function_patterns:
+        for node in gm.graph.nodes:
+            if matches_module_function_pattern(pattern, node, modules):
+                if len(node.args[0].users) > 1:  # Output of conv is used by other nodes
+                    continue
+                if len(node.args) != 8:
+                    continue
+                conv = modules[node.args[0].target]
+                if conv.training or node.args[5]:
+                    continue
+                if type(node.args[7]) is not float:
```

Comment:
suggest to assign `node.args[i]` to some variables like `bn_eps = node.args[7]` first before using them. This improves the readability.

Reply:
Changed.